src/js/player.js

```diff
@@ -2218,37 +2219,39 @@ class Player extends Component {
    *         The current video source when getting
    */
   src(source) {
-    if (source === undefined) {
```

Comment:
I really think this should be separate and at the top here. It makes a clear delineation between when it's being called as a getter vs when it's called as a setter.